apps/condo/domains/property/components/BasePropertyMapForm/index.tsx

```diff
@@ -34,6 +34,8 @@ const BasePropertyMapForm: React.FC<IPropertyMapFormProps> = (props) => {
     const PromptTitle = intl.formatMessage({ id: 'pages.condo.property.warning.modal.Title' })
     const PromptHelpMessage = intl.formatMessage({ id: 'pages.condo.property.warning.modal.HelpMessage' })
     const UnsavedChangesError = intl.formatMessage({ id: 'pages.condo.property.warning.modal.unsavedChanges' })
+    const UnitLabelsNotUniqueError = intl.formatMessage({ id: 'pages.condo.property.warning.modal.SameUnitNamesErrorMsg' })
+    const UnitLabelEmptyError = intl.formatMessage({ id: 'pages.condo.property.warning.modal.notValidUnitName' })
```

Comment:
`not valid` or `empty`? :)
```suggestion
    const UnitLabelEmptyError = intl.formatMessage({ id: 'pages.condo.property.warning.modal.emptyUnitName' })
```